docs/content/guide/component.ngdoc

```diff
@@ -420,18 +420,19 @@ The examples use the [Jasmine](http://jasmine.github.io/) testing framework.
 **Controller Test:**
 ```js
 describe('component: heroDetail', function() {
-  var component, scope, hero;
+  var component, scope, hero, componentController;
 
   beforeEach(module('simpleComponent'));
 
   beforeEach(inject(function($rootScope, $componentController) {
     scope = $rootScope.$new();
+    componentController = $componentController;
```

Comment:
It is better to keep the original name of the service.
Name the variable `$componentController` and inject it as `_$componentController_`.